modules/tensor_mechanics/test/tests/shell/static/inclined_straintest.i

```diff
@@ -0,0 +1,377 @@
+# Static test for the inclined shell element.
```

Comment:
It looks like you didn't add these tests to a `tests` file, so these won't actually get run automatically by `run_tests`. You'll also need to add results to a `gold` directory for the comparisons.